internal/promotion/promotion.go

```diff
@@ -168,6 +170,12 @@ func (s *Step) BuildEnv(promoCtx Context, opts ...StepEnvOption) map[string]any
 			"project":   promoCtx.Project,
 			"promotion": promoCtx.Promotion,
 			"stage":     promoCtx.Stage,
+			"targetFreight": map[string]any{
+				"name": promoCtx.TargetFreightRef.Name,
+				"origin": map[string]any{
+					"name": promoCtx.TargetFreightRef.Origin.Name,
+				},
+			},
```

Comment:
I am unsure if this should be here, or under `meta`.

Reply:
Agree, @johnpekcan and I talked for a bit about it (we work together) and I could see it both ways. The addition of actor under keys is good prior art for moving to since it’s info specific to a promotion (who started it, what freight was the target, etc.)

Reply:
I have pushed an additional commit to move this under `meta`. 
Please let me know if there's anything else I can do. 🍻